generator/templates/schemavalidator.gotmpl

```diff
@@ -183,32 +183,32 @@ if err := {{.ValueExpression}}.Validate(formats); err != nil {
 {{define "schemavalidator" }}{{ if .Enum }}
 // for schema
 var {{ camelize .Name }}Enum []interface{}
-func ({{ .ReceiverName }} {{ if not .IsPrimitive }}*{{ end }}{{ if .IsExported }}{{ pascalize .Name}}{{ else }}{{ .Name }}{{ end }}) validate{{ pascalize .Name }}Enum(path, location string, value {{ template "dereffedSchemaType" . }}) error {
-  if {{ camelize .Name }}Enum == nil {
-    var res []{{ template "dereffedSchemaType" . }}
-    if err := json.Unmarshal([]byte(`{{ json .Enum }}`), &res); err != nil {
-      return err
-    }
-    for _, v := range res {
-      {{ camelize .Name }}Enum = append({{ camelize .Name }}Enum, v)
-    }
+func init() {
+  var res []{{ template "dereffedSchemaType" . }}
+  if err := json.Unmarshal([]byte(`{{ json .Enum }}`), &res); err != nil {
+    return err
```

Comment:
True! I probably didn't run my tests after regenerating the bindata.go file :(

Maybe `panic(err)` would be appropriate there? If we can't initialize the validator, it's because the validator code must be wrong, and not something that client code should be able to capture and deal with?